tools/python/stylesheet/cat_stat.py

```diff
@@ -187,7 +187,7 @@ def b2t(b, text='yes'):
     for row in find_popular_taginfo(cursor, seen):
         r = ['{}={}'.format(row[0], row[1])] + ['']*6 + [row[2]]
         if wcur is not None:
-            wcur.execute("select description from wikipages where key=? and value=? and lang in ('en', 'ru') order by lang desc", (row[0], row[1]))
+            wcur.execute("select description from wikipages where key=? and value=? and lang is 'en' order by lang asc", (row[0], row[1]))
```

Comment:
Do you need to order one lang by lang? :)

Reply:
true, fixed :)